src/components/OrderFormItem.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+
+class OrderFormItem extends React.Component {
```

Comment:
this could be a stateless component, since it has no state. Also, commented out code should be removed